src/util.ts

```diff
@@ -6,19 +6,26 @@ import * as fs from 'fs';
 import {input, output} from './io';
 
 type RecordOf<T extends string> = Record<T, string | undefined>;
-export const tools = new Toolkit<RecordOf<input>, RecordOf<output>>({
-  secrets: [
-    'GITHUB_EVENT_PATH',
-    'GITHUB_EVENT_NAME',
-    'GITHUB_REF',
-    'GITHUB_ACTOR',
-  ],
-});
+let tools: Toolkit<RecordOf<input>, RecordOf<output>> | undefined;
+function getToolkit() {
+  if (!tools) {
+    tools = new Toolkit<RecordOf<input>, RecordOf<output>>({
+      secrets: [
+        'GITHUB_EVENT_PATH',
+        'GITHUB_EVENT_NAME',
+        'GITHUB_REF',
+        'GITHUB_ACTOR',
+      ],
+    });
+  }
+
+  return tools;
+}
```

Comment:
Why is this preferable to re-using the same toolkit instance?

Reply:
Cool!
I'll test this asap then!

(Sorry for the late reply, I had made this comment two weeks ago but forgot to publish it ahahhaahh)